IFC4x3/PropertyEnumerations/d/PEnum_DataTransmissionUnitType.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_DataTransmissionUnitType" Name="PEnum_DataTransmissionUnitType" UniqueId="5b09fb61-1a57-4867-ba85-842e435fff8c">
+	<Documentation></Documentation>
+	<Constants>
+		<DocPropertyConstant xsi:nil="true" href="EARTHQUAKE_13usINaH9fgQNjhblO4Co" />
+		<DocPropertyConstant xsi:nil="true" href="FOREIGNOBJECT_1ORjlzNl598e4EvrrnHH57" />
+		<DocPropertyConstant xsi:nil="true" href="WINDANDRAIN_1mtjp8YHTEevaI4iHqfaXH" />
+		<DocPropertyConstant xsi:nil="true" href="OTHER_0SAtxxJj15wQA15w2MsIV" />
+		<DocPropertyConstant xsi:nil="true" href="NOTKNOWN_0_n254Fb10hBW_0_XIj7Qe" />
+		<DocPropertyConstant xsi:nil="true" href="UNSET_01y959UgrAlvv6_W__S9Ap" />
+	</Constants>
+</DocPropertyEnumeration>
+
```

Comment:
Just out of ignorance: To me the constants seem a little bit unrelated to "DataTransmissionUnitType"? Feels more like sensor types or something.